mappings/net/minecraft/resource/ResourceReloadListener.mapping

```diff
@@ -1,9 +1,10 @@
 CLASS wz net/minecraft/resource/ResourceReloadListener
```

Comment:
You should be careful not to change any names here in yarn that would make the API make no sense. This looks fine to me, just leaving this comment here to make you aware.

Reply:
The name changes should be fine; I just renamed helper to Synchronizer (player suggestion)